src/app/_components/footer.tsx

```diff
@@ -1,46 +1,49 @@
-                    {mathAndCS.map((item) => <Link href="" className="hover:text-white/80 hover:underline-offset-2 m-1 font-light text-sm">{item}</Link>)}
-                </div>
-                <div className="w-1/6 flex flex-col flex-wrap lg:h-96 justify-left">
-                    <p>Sciences</p>
-                    {sciences.map((item) => <Link href="" className="hover:text-white/80 hover:underline-offset-2 m-1 font-light text-sm">{item}</Link>)}
-                </div>
-                <div className="w-1/6 flex flex-col flex-wrap lg:h-96 justify-left">
-                    <p>Social Sciences</p>
-                    {socialSciences.map((item) => <Link href="" className="hover:text-white/80 hover:underline-offset-2 m-1 font-light text-sm">{item}</Link>)}
-                </div>
-                <div className="w-1/6 flex flex-col flex-wrap lg:h-96 justify-left">
-                    <p>Languages</p>
-                    {language.map((item) => <Link href="" className="hover:text-white/80 hover:underline-offset-2 m-1 font-light text-sm">{item}</Link>)}
-                </div>
-                <div className="w-1/6 flex flex-col flex-wrap lg:h-96 justify-left">
-                    <p>Arts</p>
-                    {arts.map((item) => <Link href="" className="hover:text-white/80 hover:underline-offset-2 m-1 font-light text-sm">{item}</Link>)}
-                </div>
-                <div className="w-1/6 flex flex-col">
-                    [Logo] AP Students
-                    <div className="flex flex-row text-center align-center justify-center pt-8">
-                        <Instagram />
+        <div className="w-full bg-gray-900 text-center">
+            <div className="grid sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-6 min-h-24 text-center items-start justify-center my-5">
+                <Column heading="Math and CS" items={mathAndCS} />
+                <Column heading="Sciences" items={sciences} />
+                <Column heading="Social Sciences" items={socialSciences} />
+                <Column heading="Languages" items={languages} />
+                <Column heading="Arts" items={arts} />
+                <div className="flex flex-col gap-4 self-center justify-self-center mt-3">
+                    <Image className="mx-auto" src="/APStudents.png" alt="AP Students Logo" width="75" height="75" /> 
+                    <p>AP Students</p>
+                    <div className="flex flex-row">
                         <Discord />             
+                        <Instagram />
                         <Gmail />
                     </div>
                 </div>
             </div>
             <hr className="h-px bg-gray-200 border-0 dark:bg-gray-700" />
-            <p className="text-sm py-6">&#169; Copyright 2024 - AP Students</p>
+            <p className="text-sm py-6">&#169; Copyright {new Date().getFullYear()} - AP Students</p>
+        </div>
+    )
+}
+
+interface Props {
+    heading: string;
+    items: any;
```

Comment:
this should be a more descriptive type (and I would rename `Props` to `ColumnProps`